scripts/bootstrap-pallets/src/bootstrap/assets/index.ts

```diff
@@ -0,0 +1,10 @@
+import { mintAssetsToAddress } from "@bootstrap-pallets/lib";
+import { ApiPromise } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+
+export async function bootstrapAssets(api: ApiPromise, walletSudo: KeyringPair, mint: [string, string, string][]): Promise<void> {
```

Comment:
nit: would love a struct for the mint function

Reply:
@KaiserKarel you mean object with named keys?